governance/fma-isthmus-l2-pectra.md

```diff
@@ -0,0 +1,195 @@
+
+# Pectra Features on Isthmus
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Failure Modes and Recovery Paths](#failure-modes-and-recovery-paths)
```

Comment:
One failure mode I don't see is that L1 Pectra means `msg.sender == tx.origin` checks no longer ensure the caller has no code. We have this check in the PreimageOracle, and we have `Address.isContract` checks in the ERC721Bridges and StandardBridges. The latter two are codesize based so I believe they should be unaffected. But we should have a failure mode for this, and explain why (hopefully) this change doesn't break anything

Reply:
Asked in discord about this as well: https://discord.com/channels/1244729134312198194/1331301843829002313/1342155969400012860

Reply:
Good call out! Will add a section for that

Reply:
Update on the plan for these checks is here: https://github.com/ethereum-optimism/design-docs/pull/209